distributed-calculator/README.md

```diff
@@ -658,6 +658,16 @@ Our client persists state by simply POSTing JSON key-value pairs (see `react-cal
     });
 ```
 
+## [Optional Steps] VS Code Debugging
+
+Leveraging VS Code you can manually configure your launch.json and task.json to launch Dapr and your applications into debug mode.
```

Comment:
Since the files are already included for you, you don't really need to manually configure anything (correct?) I would rephrase to something like 

"If you are using Visual Studio Code, you can debug this application using preconfigured `launch.json` and `task.json` files in the `.vscode` folder... "